sync/src/tests/inflight_blocks.rs

```diff
@@ -20,9 +21,11 @@ fn inflight_blocks_count() {
     assert_eq!(inflight_blocks.peer_inflight_count(1.into()), 2);
     assert_eq!(inflight_blocks.peer_inflight_count(2.into()), 1); // one block inflight
     assert_eq!(
-        inflight_blocks
-            .inflight_block_by_peer(1.into())
-            .map(|set| set.iter().collect()),
+        inflight_blocks.inflight_block_by_peer(1.into()).map(|set| {
+            let mut res = set.iter().collect::<Vec<&H256>>();
+            res.sort();
+            res
+        }),
```

Comment:
I think to compare the `HashSet` is more reasonable.

Reply:
`vec + sort` or `HashSet` both ok. But `HashSet` needs more code, so ...

Reply:
reasonable and less code

```rust
    use fnv::FnvHashSet;
    use std::iter::FromIterator;

    assert_eq!(
        inflight_blocks
            .inflight_state_by_block(&h256!("0x4"))
            .map(|state| state.peers.clone()),
        Some(FnvHashSet::from_iter(vec![(4.into()), (1.into())]))
    );
```